shared/cfg/guest-os/Linux/RHEL/8/8.devel.cfg

```diff
@@ -0,0 +1,9 @@
+- devel:
+    # Uses defaults set in 8.cfg
+    # Adding x.y versions should only require changing the kickstart
+    # and adding per-arch image shas.
+    image_name = images/${os_variant}devel-${vm_arch_name}
+    unattended_install.cdrom:
+        cdrom_cd1 = isos/linux/RHEL-8-devel-${vm_arch_name}.iso
+    unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install:
+        unattended_file = unattended/RHEL-8-devel.ks
```

Comment:
Well at this point the KS file does not exists. So ideally it should be just one patch...